lib/sidekiq/superworker/subjob.rb

```diff
@@ -86,15 +84,44 @@ def jid(superjob_id, subjob_id)
         def redis_prefix
           Superworker.options[:subjob_redis_prefix]
         end
+
+        def create_redis_multi_pipeline(&block)
```

Comment:
Should this be private if the only access is via transaction? That would prevent someone from accidentally calling it when they should be using `redis_connection_transaction`?

Reply:
Good idea!